codec-http/src/main/java/io/netty/handler/codec/http/HttpContentEncoder.java

```diff
@@ -67,7 +67,7 @@
 
     private final Queue<CharSequence> acceptEncodingQueue = new ArrayDeque<CharSequence>();
     private EmbeddedChannel encoder;
-    private State state = State.AWAIT_HEADERS;
+    private volatile State state = State.AWAIT_HEADERS;
```

Comment:
This change makes no sense as all is called from a single thread

Reply:
Sorry,I've done it